src/modules/taskStatus.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable arrow-body-style */
+
+import { storeTask, render } from './functions.js';
```

Comment:
Please try to find a way to solve the linter errors than disabling them. They are meant to teach you how to use the best practices when writing code no matter how illogical they may seem sometimes.

Reply:
I got you @KDlamini